src/main/java/nextstep/ladder/domain/ladder/Result.java

```diff
@@ -0,0 +1,33 @@
+package nextstep.ladder.domain.ladder;
+
+public class Result {
+    private static final int MAX_LENGTH = 5;
+    private static final String NULL_BLANK_EXCEPTION_MESSAGE = "사다리 결과는 Null 또는 공백 문자열을 허용하지 않습니다.";
+
+    private final String value;
+
+    public Result(String value) {
+        if (isNullOrEmpty(value)) {
+            throw new IllegalArgumentException(NULL_BLANK_EXCEPTION_MESSAGE);
+        }
+
+        if (isOverSize(value)) {
+            throw new IllegalArgumentException("결과 값은 " + MAX_LENGTH + " 길이 이하이어야 합니다.");
+        }
+
+
+        this.value = value.strip();
+    }
```

Comment:
```suggestion

        this.value = value.strip();
    }
```

공백 한 줄은 빼도 될 것 같네요 😎